server/services/riskQuery.js

```diff
@@ -122,7 +185,19 @@ async function externalQueries (x, y, queries) {
     }
     results.forEach((result, index) => {
       if (queries[index].esriCall) {
-        featureLayers[queries[index].key] = result.features
+        if (!((result.features) || (result.layers))) {
+          console.log('Error: Invalid response for %s', queries[index].key)
+          console.log(result)
```

Comment:
Just checking if these logs are meant to be here.

Reply:
Yes, this one is to log the data that caused the issue in the error logs, as an error is thrown because of this.